tests/kernel/timer/timer_behavior/pytest/test_timer.py

```diff
@@ -52,6 +52,29 @@ def do_analysys(test, stats, config, sys_clock_hw_cycles_per_sec):
     logger.info(f'real drift:     {time_diff * 1_000_000:.6f} us')
     logger.info('====================================================')
 
+    logger.info('RECORD: {'
+                f'"testcase":"jitter_drift_timer"'
```

Comment:
would be easier to create json with json lib and use just json.dumps method

Reply:
isn't it much easier just to print ? there is no need to parse json here.

Reply:
@gchwier, changed from print to json.dumps